src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +11,11 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+        if (userService.findByEmail(email) != null) {
+            return userService.findByEmail(email).getPassword().equals(password);
```

Comment:
save userService.findByEmail(email) for variable and reuse it, simplify code to one line after return

Reply:
I tried to fix it, I’m not sure I understood your idea correctly

Reply:
please tell me what needs to be fixed?